INSTALL

```diff
@@ -17,7 +17,7 @@ To run a Cache, you will need to:
 	1. customize the squid.conf configuration file:
 	      % vi /usr/local/squid/etc/squid.conf
 
-	2. Initalise the cache:
+	2. Initialise the cache:
```

Comment:
@mrumph, is there a way to tell codespell to prefer one spelling over the other? This is not critical or even important, but most Squid sources use the (American) "initialize" variant rather than the (British) "initialise" variant. I do not think codespell should change one for the other (or, at the very least, that deserves a separate discussion that I do not want to have now/here), but when fixing a spelling mistake, can we tell codespell to use the prevailing (in Squid) variant?

```shell
$ git grep -i initialise | wc -l
88
$ git grep -i initialize | wc -l
416
```

Reply:
One possibility (but not really desirable for a number of reasons) is to supply our own dictionary with the -D option on codespell.

Reply:
If that custom dictionary will _replace_ the standard one (rather than just overwrite a few entries), then I agree that `-D` is not a good solution. Let's ignore this problem.